lms/templates/courseware/courseware.html

```diff
@@ -231,3 +231,5 @@
 </nav>
 
 <%include file="../modal/accessible_confirm.html" />
+
+<%include file="../stanford/_course_ga_tag.html" args="course_ga_tag=course_ga_tag" />
```

Comment:
I'd prefer to wrap the conditional around the `include` itself;
why include an empty file if we can avoid it?